frontend/src/pages/WhyToTeachPage/WhyToTeachPage.tsx

```diff
@@ -163,7 +163,7 @@ const HowTeachingProgramWorks: React.FC = () => (
         vyučování, být bezúhonní, <strong>zdravotně způsobilí</strong> a mít{' '}
         <strong>odbornou kvalifikaci</strong> pro konkrétní přímou pedagogickou činnost. Pro
         přehledné shrnutí,{' '}
-        <Button href={`${routes.canITeach}#zakon-o-pedagogickych-pracovnicich`} buttonStyle="link">
+        <Button buttonStyle="link" css={css`display: inline`} href={`${routes.canITeach}#zakon-o-pedagogickych-pracovnicich`}>
```

Comment:
dal bych to klidně k Link do `Button/styled.ts`. Nebo tě napadá místo kde by mohlo vadit mít to globálně?

Reply:
a nebo jestli to necháš takhle, tak to chce opravit tu chybu. Mě to hlásí, že prop `css` na komponentě `Button` neexistuje

Reply:
Mas pravdu, gobalne to bude lepsi. Opraveno.